lib/manual_relocator.rb

```diff
@@ -134,7 +134,7 @@ def reslug
       sections.each do |section|
         reslug_msg = "Reslugging section '#{section.slug}'"
         new_section_slug = section.slug.gsub(from_slug, to_slug)
-        puts "#{reslug_msg} as '#{section.slug}'"
+        puts "#{reslug_msg} as '#{new_section_slug}'"
```

Comment:
Minor: I might be tempted to inline `reslug_msg`. If you did that as a refactoring before your change, it would make it more obvious that there is a problem.